numpy/random/mtrand/mtrand.pyx

```diff
@@ -3817,7 +3817,7 @@ cdef class RandomState:
         Draw samples from a negative binomial distribution.
 
         Samples are drawn from a negative binomial distribution with specified
-        parameters, `n` trials and `p` probability of success where `n` is an
+        parameters, `n` successes and `p` probability of success where `n` is an
```

Comment:
The word `trials` here was incorrect in literature. `trials = successes + failures`, see the original issue for why.